packages/cohttp-async/cohttp-async.1.0.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: "anil@recoil.org"
+authors: [
+  "Anil Madhavapeddy"
+  "Stefano Zacchiroli"
+  "David Sheets"
+  "Thomas Gazagnaire"
+  "David Scott"
+  "Rudi Grinberg"
+  "Andy Ray"
+]
+homepage: "https://github.com/mirage/ocaml-cohttp"
+bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
+license: "ISC"
+tags: ["org:mirage" "org:xapi-project"]
+dev-repo: "https://github.com/mirage/ocaml-cohttp.git"
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "cohttp"
```

Comment:
Should each of these `cohttp*` dependencies in each of the new packages have a version constraint to `>= "1.0.0"`?

Reply:
I think adding such constraints would be pretty conservative and would invalidate some functional build plans. So I would say I'm against adding such constraints unless we have concrete build failures on our hands.

Reply:
That's fine by me as long as it can build against older cohttp base packages.